### PR TITLE
min build whitelisting

### PR DESCRIPTION
**Pull request by jdevcs**
## Description

https://github.com/web3/web3.js/issues/5743

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: packages/web3-validator/package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
 	},
 	"files": [
 		"lib/**/*",
-		"src/**/*"
+		"src/**/*",
+		"dist/**/*"
```
Comment (jdevcs) [inline packages/web3-validator/package.json:25]: After a PR in past where lib and dist both directories are created for outputs, dist was not whitelisted, so due to current CDN structure web3.min is not available in last few RCs as its not uploaded to npm due to filtering, 

Its fixed here. 

tested `dist/min..` including via` yarn pack`

**Review by avkos — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Review by Sammiegirl411 — COMMENTED** [review]
Ok

**Review by Sammiegirl411 — COMMENTED** [review]
Ok

**Review by Sammiegirl411 — COMMENTED** [review]
Ok

**Pull request merged**